pkg/ottl/contexts/internal/span.go

```diff
@@ -571,3 +577,28 @@ func accessStatusMessage[K SpanContext]() ottl.StandardGetSetter[K] {
 		},
 	}
 }
+
+func accessAdjustedCount[K SpanContext]() ottl.StandardGetSetter[K] {
```

Comment:
Could you add tests for the code paths where we can't get the trace state? (can't be parsed, missing, etc.)

Reply:
Looks great, thank you. Really appreciate the comments for context.